includes/class-wc-stripe-helper.php

```
@@ -1114,14 +1114,55 @@ public static function has_cart_or_checkout_on_current_page() {
+		if ( WC_Stripe_Country_Code::INDIA === strtoupper( $account_country ) ) {
+			return 'account-country';
+		}
+
+		// If we are in test mode, payout details are often missing and currency-based rules
+		// are not enforced.
+		if ( WC_Stripe_Mode::is_test() ) {
+			return null;
+		}
+
+		// Check that the store currency can be used for settlement.
+		$stripe_settlement_currencies = $stripe_account->get_supported_store_currencies();
```

PHPMD (2.15.0)

[warning] Avoid excessively long variable names like $stripe_settlement_currencies. Keep variable name length under 20. (undefined)

(LongVariable)